### ISSUE TITLE
Status?  Is this still alive?

### ISSUE THREAD

**Issue by Nanoc-ice** [vaughantay/roguelove#2]
Just wondering about the status of this project.  Is it complete or at least useable?  If not complete, what is outstanding that a dev would need to know about before using this framework?

**Comment by vaughantay** [comment]
Sorry for the late response, I didn't notice this had been posted until now, need to figure out how to get GitHub to alert me when someone posts an issue.
It's definitely still alive and slowly being worked on. I wouldn't call it "complete" but you could definitely make a game with it, although it may be hard to use for people who aren't as familiar with it as I am because there's not much documentation. 
What little documentation there is can be found at https://docs.google.com/document/d/1bJmuokbK8Xtd2P9K8KRQRSeGdHd78HGKuOKaZltCoE4/edit and in the doc/ directory of this repository. And if you do poke around with it and have questions, I'm happy to answer them.

From a gameplay perspective, everything works fine. All the gameplay systems like map generation, creatures, items, special abilities, special terrain, factions, missions, stores, random events, etc. are all totally usable. Graphics, sound, and music systems are in place. There are more features I want to add (you can see them at https://trello.com/c/EhSWsFKY/) but it's mostly just extras on top of the base systems, the base systems themselves are all done.
There is a "demo" game that comes with it though it's not really a complete game, mainly just a bunch of content I've been using to test out various things that showcases a lot of what's possible.

Aside from the lack of documentation, I think the biggest issue is some UI work that needs to be done. Many UI screens haven't been updated to work with variable font sizes. And there are a couple of features (namely faction favor and mission status) that aren't displayed anywhere for the player to access. Oh, and the screen that displays gameplay stats is completely broken.

tl;dr Theoretically it's totally usable, but not really in as easy-to-use of a state as I'd like yet.

**Comment by Nanoc-ice** [comment]
This sounds like great news and provides a solid framework to start.  I can understand being a little rough around the edges, but if you come this far with it, I bet they will get smoothed out in time.  It will provide a great jump start on my project (undefined at the moment, other then roguelike), so that I am not starting from a blank canvas!  I will let you know if/when I run into hurdles I can’t clear.

Your work on this framework is very much appreciated and thanks for the reply.
